x-pack/plugins/endpoint/server/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { EndpointPlugin } from './plugin';
+
+export const config = {
+  schema: schema.object({ enabled: schema.boolean({ defaultValue: false }) }),
+};
+
+export function plugin() {
```

Comment:
JFYI [PluginInitializerContext](https://github.com/elastic/kibana/blob/master/docs/development/core/server/kibana-plugin-server.plugininitializercontext.md) also passed to the plugin factory
```ts
export const plugin = (initializerContext: PluginInitializerContext) => new EndpointPlugin(initializerContext)
```